device-types/Arista/AWE-5310.yaml

```diff
@@ -0,0 +1,37 @@
+---
+manufacturer: Arista
+model: AWE-5310
+slug: arista-awe-5310
+part_number: AWE-5310
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c16
+    maximum_draw: 550
+  - name: PS2
+    type: iec-60320-c16
+    maximum_draw: 550
```

Comment:
It appears that these PSUs are modular. If this is the case, then please move the power to `module-bays` and create the associated modular PSUs within the respective module-types folder